app/src/main/java/org/jellyfin/androidtv/ui/shared/KeyboardFocusChangeListener.kt

```diff
@@ -0,0 +1,16 @@
+package org.jellyfin.androidtv.ui.shared
+
+import android.content.Context
+import android.view.View
+import android.view.inputmethod.InputMethodManager
+
+class KeyboardFocusChangeListener : View.OnFocusChangeListener {
+	override fun onFocusChange(v : View?, hasFocus : Boolean) {
+		if (v != null) {
```

Comment:
```suggestion
	override fun onFocusChange(v : View, hasFocus : Boolean) {
```

If the docs are clear that it won't be null you can just remove the nullability.

Reply:
I blame Android Studio for adding the `?` there. :wink: